smart-contracts/package.json

```diff
@@ -17,12 +17,9 @@
     "eslint": "^5.7.0",
     "ethereumjs-units": "^0.2.0",
     "openzeppelin-eth": "^2.0.2",
-    "shelljs": "^0.8.3",
     "truffle": "^4.1.14",
     "truffle-contract": "^3.0.6",
-    "web3-utils": "^1.0.0-beta.36",
-    "zos": "^v2.1.0-rc.0",
-    "zos-lib": "^2.0.2"
```

Comment:
Zos-lib is where we import `Initializable.sol`, so I think we need it to compile.

Reply:
Something fishy is going on then because build passed :(

Reply:
Something fishy is going on then because build passed :(